crates/wasmtime/src/config.rs

```diff
@@ -1478,6 +1482,15 @@ impl Config {
         self
     }
 
+    /// Enables memory error checking for wasm programs.
+    ///
+    /// This option is disabled by default.
+    pub fn wmemcheck(&mut self, enable: bool) -> &mut Self {
```

Comment:
Could this perhaps cause an error to get returned during `Engine::new` if `wmemcheck` wasn't enabled at compile time?

Reply:
Yup, added a check to `Config::validate()`!